test/parallel/test-crypto-dh.js

```diff
@@ -182,4 +182,78 @@ ecdh4.setPublicKey(ecdh1.getPublicKey());
 
```

Comment:
Since we're in this test making changes, could I ask you to go ahead and use `const` for the require decls at the top?